browser/autocomplete/brave_autocomplete_provider_client.cc

```diff
@@ -1,33 +1,21 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 #include "brave/browser/autocomplete/brave_autocomplete_provider_client.h"
 
 #include "base/strings/utf_string_conversions.h"
 #include "brave/common/webui_url_constants.h"
-#include "chrome/browser/profiles/profile.h"
-#include "chrome/browser/search_engines/template_url_service_factory.h"
 #include "chrome/common/webui_url_constants.h"
 
 BraveAutocompleteProviderClient::BraveAutocompleteProviderClient(
-    Profile* profile)
-    : ChromeAutocompleteProviderClient(profile->GetOriginalProfile()),
```

Comment:
I'm fairly certain we're leaking private autocomplete into the regular profile here in the old code and here in the updated code https://github.com/brave/brave-core/pull/5221/files#diff-2325211a427ac6f007ad338411e100aaR14 cc @diracdeltas